geps/gep-1494/index.md

```diff
@@ -0,0 +1,144 @@
+
+Basic auth is defined in [RFC-7617](https://datatracker.ietf.org/doc/html/rfc7617).
+
+#### TLS Client Certificate Authentication
+
+TLS includes the possibility of having both the client and server present certificates for the other party to validate. (This is often called "mutual TLS", but is distinct from the use of that term in Service Mesh contexts, where it means something more like "mutual TLS with short-lifetime, automatically created and managed dynamic keypairs for both client and server").
+
+In this case, the server authenticates the client based on the client presenting a certificate that's signed by an authority that's also trusted by the server's trust chain. Some implementations also allow details about the certificate to be passed through to backend clients, to be used in authorization decisions.
+
+TLS v1.3 is defined in [RFC-8446](https://datatracker.ietf.org/doc/html/rfc8446), with v1.2 defined in [RFC-5246](https://datatracker.ietf.org/doc/html/rfc5246).
+
+#### JWT
+
+JWT is a format for representing _claims_, which are stored as fields in a JSON object. In common use, these claims represent various parameters that describe the properties of an authentication, with sample claims being `iss`, Issuer, `sub`, Subject, and `aud`, Audience.
+
+JWT also specified ways in which JWTs can be _nested_, where a JWT is encoded inside another JWT, which allows the use of encryption and signing algorithms. In the Gateway API Auth* case, this is expected to be common, in a pattern looking something like this:
+
+- a common authority issues keypairs to clients and servers
+- clients include JWTs with requests that identify themselves and their intended uses
+- servers unwrap encryption and validate signatures of these JWTs to validate the chain of trust
+- servers use the authentication details to make authorization decisions.
+
+JWT is defined in [RFC-75199](https://datatracker.ietf.org/doc/html/rfc7519).
+
+#### Oauth2 and OIDC
+
+Oauth2 is an _authorization framework_, which allows clients and servers to define ways to perform authentication and authorization in as secure a way as possible. It extensively uses TLS for encryption, and involves a third-party handling the authorization handshake with a client, which the third-party then provides to the server.
+
+Open ID Conect (OIDC) is a protocol based on the OAuth 2 framework, that allows Users to talk to Identity Providers (IDPs), on behalf of a Relying Party (RP), and have the IDP give the user an Identity Token (which the User's browser can then provide as Authentication to the Relying Party), and also allows the RP to request Claims about the User, which can be used for Authorization.
+
+
+## Auth* User Stories
+
+
+* As Ana the Application Developer, I wish to be able to configure that some or all of my service exposed via Gateway API requires Authentication, and ideally to be able to make Authorization decisions about _which_ authenticated clients are allowed to access.
```

Comment:
This might leak into the "API" phase, but I think there's 2 levels to this, which are both worth an explicit mention

* The API proposed implementation provides enough flexibility to integrate with an authorization mechanism and protect resources entirely in the gateway
* The API allows to inject information about the authentication result into the requests and allows bakend application to make authorization decisions based on this.